xtra-libp2p/src/dialer.rs

```diff
@@ -47,9 +47,9 @@ impl Actor {
     }
 
     fn stop_with_error(&mut self, e: Error, ctx: &mut xtra::Context<Self>) {
-        tracing::debug!("Stopping dialer with an error: {e:#}");
+        tracing::debug!("Stopping dialer with an error: {e:?}");
```

Comment:
did you mean to change this line? it probably doesn't belong to this changeset :)